server/app/core/config.py

```
@@ -0,0 +1,15 @@
+from pydantic_settings import BaseSettings
+from typing import List
+
+class Settings(BaseSettings):
+    DATABASE_URL: str = "sqlite:///./sky_casa_web.db"
+    JWT_SECRET: str = "change-me"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "JWT_SECRET"

(S105)